tests/e2e.bats

```diff
@@ -1,5 +1,24 @@
 #!/usr/bin/env bats
 
+# Define global variables (for use in teardown)
+setup() {
+  namespace="kube-system"
+  ds_name="dranet"
+  node_selector_key="disable"
```

Comment:
Lets keep things simple and avoid defining variables unless they are absolutely reused (`node_selector_key` and `node_selector_value` as an example)